### PR TITLE
Translate "Ruby 3.3.5 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translates #3359, #3360

Actual diff is 697873500c98348e17ac5f06d32512a0620a317c

### CODE REVIEW

File: ko/news/_posts/2024-09-03-3-3-5-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "Ruby 3.3.5 릴리스"
+author: k0kubun
+translator: shia
+date: 2024-09-03 06:40:00 +0000
+lang: ko
+---
+
+Ruby 3.3.5가 릴리스되었습니다.
+
+이번 릴리스에는 일반적인 업데이트로, 사소한 버그 수정이 포함되어 있습니다.
```
Comment (yous) [inline ko/news/_posts/2024-09-03-3-3-5-released.md:12]:
```suggestion
이번 릴리스는 일반적인 업데이트로, 사소한 버그 수정이 포함되어 있습니다.
```

File: ko/news/_posts/2024-09-03-3-3-5-released.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+가능한 빨리 Ruby 버전을 업그레이드하는 것을 권장합니다.
+자세한 내용은 [GitHub 릴리스](https://github.com/ruby/ruby/releases/tag/v3_3_5)를 참조하세요.
```
Comment (yous) [inline ko/news/_posts/2024-09-03-3-3-5-released.md:14]:
```suggestion
자세한 내용은 [GitHub 릴리스 노트](https://github.com/ruby/ruby/releases/tag/v3_3_5)를 참조하세요.
```

File: ko/news/_posts/2024-09-03-3-3-5-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-09-03-3-3-5-released.md]: fixed 3274f2b

File: ko/news/_posts/2024-09-03-3-3-5-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-09-03-3-3-5-released.md]: fixed 3274f2b

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!

**Pull request merged**